examples/kubernetes/dynamic_provisioning_withdeploymenttype/README.md

```diff
@@ -0,0 +1,80 @@
+## Dynamic Provisioning with DeploymentType
+This example shows how to create a FSx for Lustre filesystem using persistence volume claim (PVC) with DeploymentType specified
+
+### Edit [StorageClass](./specs/storageclass.yaml)
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-sc
+provisioner: fsx.csi.aws.com
+parameters:
+  subnetId: subnet-0cd1914f0b9041ebe
+  securityGroupIds: sg-07662e27978b3b360,sg-057948b58dd38e68b
+  s3ImportPath: s3://evanzempty
+  s3ExportPath: s3://evanzempty
+  deploymentType: SCRATCH_2
+mountOptions:
+  - flock,noatime
+```
+* subnetId - the subnet ID that the FSx for Lustre filesystem should be created inside.
+* securityGroupIds - a comman separated list of security group IDs that should be attached to the filesystem
+* s3ImportPath(Optional) - S3 data repository you want to copy from S3 to persistent volume
+* s3ExportPath(Optional) - S3 data repository you want to export new or modified files from persistent volume to S3
+* deploymentType: FSx for lustre now provides three deployment types which uses different EC2 instances types
+
+Note:
+- S3 Bucket in s3ImportPath and s3ExportPath must be same, otherwise the driver can not create FSx for lustre successfully.
+- s3ImportPath can stand alone and a random path will be created automatically like `s3://ml-training-data-000/FSxLustre20190308T012310Z`
+- s3ExportPath can not be given without specifying S3ImportPath.
+
+### Edit [Persistent Volume Claim Spec](./specs/claim.yaml)
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fsx-claim
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: fsx-sc
+  resources:
+    requests:
+      storage: 1200Gi
+```
+Update `spec.resource.requests.storage` with the storage capacity to request. For deployment SCRATCH_2, The storage capacity value will be rounded up to 1200 GiB, 2400 GiB, or a multiple of 2400 GiB.
+
+### Deploy the Application
+Create PVC, storageclass and the pod that consumes the PV:
+```sh
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning_withdeploymenttype/specs/storageclass.yaml
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning_withdeploymenttype/specs/claim.yaml
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning_withdeploymenttype/specs/pod.yaml
+```
+
+### Use Case 1: Access S3 files from Lustre filesystem
```

Comment:
the following is a copy & paster from the s3 example, since we have similar example there already, there is no need to duplicate the use cases but instead we should focus on the use case of deployment type itself here

Reply:
For deploymentType itself, there is not much to say here since from customer point of view, the usage is the same.

Reply:
if it's similar, maybe we can just edit the existing example? It's basically one new optional parameter/line in the storageclass, if it doesn't change customer usage/workflow then it should be fine

Reply:
Ok, I will make the change to modify the existing examples.